operations/mimir-mixin/alerts/blocks.libsonnet

```
@@ -209,6 +209,20 @@
             message: '%(product)s Store Gateway %(alert_instance_variable)s in %(alert_aggregation_variables)s has not successfully synched the bucket since {{ $value | humanizeDuration }}.' % $._config,
           },
         },
+        {
+          // Alert if the store-gateway is not owning any blocks
+          alert: $.alertName('StoreGatewayNoSyncedBlocks'),
```

I would rename it to `StoreGatewayNoSyncedTenants` because what we actually check is whether the store-gateway owns any tenant, not blocks.

```suggestion
          alert: $.alertName('StoreGatewayNoSyncedTenants'),
```

---

changed in ea7d11ff1b7dc7ac002ce4d19cdae8367e31625b